api/openapi.yaml

```diff
@@ -2171,6 +2368,11 @@ components:
               $ref: '#/components/schemas/AssetScanStats'
             summary:
               $ref: '#/components/schemas/ScanFindingsSummary'
+            providerIDs:
+              type: array
+              description: List of provider IDs which discover the asset.
+              items:
+                type: string
```

Comment:
Similarly I think this should be an array of ProviderRelationships, which will allow them to be $expand-able in the API.